### PR TITLE
MD-3028: Adding username parameter for sqlline connect

### PR DESCRIPTION
**Pull request by prasadns14**
@Agirish @rhou1 please review

### CODE REVIEW

File: framework/resources/Datasources/ctas/create_tables.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ hadoop fs -mkdir /drill/testdata/ctas/parquet
 
 if [ -z "$PASSWORD" ]
 then
-  ${DRILL_HOME}/bin/sqlline -u "jdbc:drill:schema=dfs.ctas_parquet;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}"  --run=${DRILL_TEST_DATA_DIR}/Datasources/ctas/create_tables_parquet.ddl
+  ${DRILL_HOME}/bin/sqlline -n ${USERNAME} -u "jdbc:drill:schema=dfs.ctas_parquet;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}" --run=${DRILL_TEST_DATA_DIR}/Datasources/ctas/create_tables_parquet.ddl
```
Comment (rhou1) [inline framework/resources/Datasources/ctas/create_tables.sh]: +1 LGTM

File: framework/resources/Datasources/ctas_auto_partition/setup.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ hadoop fs -mkdir /drill/testdata/ctas_auto_partition/tpch_single_partition
 
 if [ -z "$PASSWORD" ]
 then
-  ${DRILL_HOME}/bin/sqlline -u "jdbc:drill:schema=dfs.ctasAutoPartition;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}"  --run=${DRILL_TEST_DATA_DIR}/Datasources/ctas_auto_partition/create_tables.ddl
+  ${DRILL_HOME}/bin/sqlline -n ${USERNAME} -u "jdbc:drill:schema=dfs.ctasAutoPartition;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}" --run=${DRILL_TEST_DATA_DIR}/Datasources/ctas_auto_partition/create_tables.ddl
```
Comment (rhou1) [inline framework/resources/Datasources/ctas_auto_partition/setup.sh]: +1 LGTM

File: framework/resources/Datasources/limit0/aggregates/aggregation/group_by_case/create_groupByCase_views.sh [hunk 3]
```diff
@@ -6,7 +6,7 @@ set -e
 
 if [ -z "$PASSWORD" ]
 then
-  ${DRILL_HOME}/bin/sqlline -u "jdbc:drill:schema=dfs.aggregation;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}"  --run=${DRILL_TEST_DATA_DIR}/Datasources/limit0/aggregates/aggregation/group_by_case/create_groupByCase_views.sql
+  ${DRILL_HOME}/bin/sqlline -n ${USERNAME} -u "jdbc:drill:schema=dfs.aggregation;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}" --run=${DRILL_TEST_DATA_DIR}/Datasources/limit0/aggregates/aggregation/group_by_case/create_groupByCase_views.sql
```
Comment (rhou1) [inline framework/resources/Datasources/limit0/aggregates/aggregation/group_by_case/create_groupByCase_views.sh]: +1 LGTM

**Comment by Agirish** [comment]
+1

**Pull request merged**